client/app/pages/queries/queries-search-results-page.js

```diff
@@ -23,6 +23,10 @@ function QuerySearchCtrl($location, $filter, currentUser, Events, Query) {
     this.paginator.updateRows(queries);
   });
 
+  this.createdAtSort = row => row.created_at.valueOf();
+  this.createdBySort = row => row.user.name;
+  this.scheduleSort = row => row.schedule && parseInt(row.schedule, 10);
```

Comment:
I'm not sure if it's a huge deal, but schedule might be an hour as well, in which case it will get random sort for minutes (i.e. 12:30 might come before 12:00).

Reply:
-- aha, true. We haven't used that feature so I didn't notice. Should be easy to fix